src/test/java/org/folio/sidecar/integration/keycloak/filter/KeycloakAuthorizationFilterTest.java

```diff
@@ -420,4 +415,8 @@ private static String userTokenCacheKey(String userId, String sid) {
   private static String systemTokenCacheKey() {
     return String.format("%s#%s#%s", KC_PERMISSION, TENANT_NAME, SYSTEM_TOKEN_EXPIRATION_TIME);
   }
+
+  protected ModuleBootstrapEndpoint moduleBootstrapEndpoint() {
```

Comment:
Is this method fit enough for `AbstractFilterTest`?

Reply:
We already have it in AbstractFilterTest as this:
```
 protected ModuleBootstrapEndpoint moduleBootstrapEndpoint() {
    return new ModuleBootstrapEndpoint("/foo/items", "GET");
  }
```

But this particular test overrides it with a different endpoint